src/plugins/intel_npu/tests/functional/behavior/npuw/behavior_tests.hpp

```diff
@@ -0,0 +1,51 @@
+// Copyright (C) 2024 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+#pragma once
+
+#include <memory>
+
+#include <gtest/gtest.h>
+
+#include "openvino/runtime/core.hpp"
+#include "openvino/runtime/iplugin.hpp"
+#include "openvino/runtime/make_tensor.hpp"
+
+#include "mocks/mock_plugins.hpp"
+#include "mocks/register_in_ov.hpp"
+
+namespace ov {
+namespace npuw {
+namespace tests {
+
+class BehaviorTestsNPUW : public ::testing::Test {
+public:
+    ov::Core core;
+    std::shared_ptr<MockNpuPlugin> npu_plugin;
+    std::shared_ptr<MockCpuPlugin> cpu_plugin;
```

Comment:
So I'd recommend renaming these two to `mock_npu_plugin` / `mock_cpu_plugin` to avoid disambiguation.

Reply:
Thanks, will do! Will update in the following PR!